woocommerce/woocommerce-android
# [Shipping Label Revamp] Add Shipping Label status observer in Purchased UI

Summary
==========
Fix issue #12707 by adding the Shipping Label Status observation to allow the Printing action to only be available once the Label is available.

Screen Capture
==========
https://github.com/user-attachments/assets/72e19269-2eba-493c-bebb-2b5f9fbb58b8

How to Test
==========
1. Open the main Shipping Label screen
2. Complete the Shipping Label creation form
3. Hit the Purchase button
4. Once in the Purchased Screen, verify that the Shipping data is correctly presented, with a waiting message at the top.
5. Wait for the Purchase to be completed and verify the Label can be printed successfully.

Update release notes:


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
